### PR TITLE
I apologise for nothing

### PR DESCRIPTION
**Pull request by TomGranot**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -310,5 +310,7 @@
   {
     "pun": "Q: What does a programmer say after a road trip across California?",
     "punchline": "A: It was a solid state drive"
-  }
+  },
+    "pun": "Q: What do you call a programmer who acts in a strange way?",
+    "punchline:" "A: A programmer with undefined behaviour"
```
Comment (AlexLakatos) [inline puns.json:315]: It looks like you're missing a closing bracket here.

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
This made me laugh! Thanks for the PR! I've only got a minor request, it looks like you forgot a closing bracket.

**Comment by TomGranot** [comment]
@AlexLakatos This is why no one should write JSON by hand. In a web interface. On a Monday.
Fixed.

File: puns.json [hunk 2]
```diff
@@ -310,5 +310,8 @@
   {
     "pun": "Q: What does a programmer say after a road trip across California?",
     "punchline": "A: It was a solid state drive"
+  },
+    "pun": "Q: What do you call a programmer who acts in a strange way?",
```
Comment (AlexLakatos) [inline puns.json:314]:
```suggestion
  {
    "pun": "Q: What do you call a programmer who acts in a strange way?",
```

**Comment by AlexLakatos** [comment]
Thanks for the PR & Happy Hacktoberfest!

**Pull request merged**